src/desktop/sse/DesktopSseClient.js

```diff
@@ -296,7 +296,9 @@ export class DesktopSseClient {
 			const req = this._getProtocolModule()
 							.request(url, {
 								method: "GET",
-								headers: {"userIds": neverNull(this._connectedSseInfo).userIds.join(",")}
+								headers: {"userIds": neverNull(this._connectedSseInfo).userIds.join(",")},
+								// this defines the timeout for the connection attempt, not for waiting for the servers response after a connection was made
+								timeout: 20000
```

Comment:
I'm not sure but I think we need read timeout?
I actually checked Android code and it has different timeout:
https://github.com/tutao/tutanota/blob/ada0750ff4bd9de41dc4558df0afbeedd0dcbce3/app-android/app/src/main/java/de/tutao/tutanota/push/PushNotificationService.java#L204-L205

so I'm not sure I suggested correctly, in the web client 20s is timeout when there's no progress so it's read timeout.